.github/workflows/linux.yml

```diff
@@ -240,7 +240,9 @@ jobs:
         run: make wasi-libc
       - run: make gen-device -j4
       - name: Test TinyGo
-        run: make ASSERT=1 test
+        run: |
+          go get tinygo.org/x/device@669b1701736c
```

Comment:
`go get` is for updating `go.mod`, I do not think that is correct. Also, with this you'll be updating the `go.mod` in the main repository, not for devices.